src/backend/agent.js

```diff
@@ -185,13 +185,15 @@ export default class Agent extends EventEmitter<{|
 
   getIDForNode(node: Object): number | null {
     for (let rendererID in this._rendererInterfaces) {
-      // A renderer will throw if it can't find a fiber for the specified node.
-      try {
-        const renderer = ((this._rendererInterfaces[
-          (rendererID: any)
-        ]: any): RendererInterface);
-        return renderer.getFiberIDForNative(node, true);
-      } catch (e) {}
```

Comment:
Ah, nice find with the `null` check!

Although I wonder if older React versions might throw? This comment was based on this one:
https://github.com/facebook/react-devtools/blob/d839081f79f681617df7584a8ffd2f4163fa40b9/agent/Agent.js#L310-L313

Looks like error handling was added in [3979c81](https://github.com/facebook/react-devtools/commit/3979c811d5cf19da9924a9aff33fb1484adb6a0d#diff-7d7518fc7bed10b7a64646d6b363b99aR225) and the comment was later added in [11537a0](https://github.com/facebook/react-devtools/commit/11537a04ffda71b0c75ca5346354d0df49a3bc89#diff-7d7518fc7bed10b7a64646d6b363b99aR269). While DevTools itself doesn't throw, maybe e.g. `nativeTagToRootNodeID` did for some versions?

Might be best to re-add the `try`/`catch` too, just in case. I can do that after merging!

Reply:
Makes sense, thanks! 🙂